docs/pages/api-docs/data-grid/grid-col-def.md

```diff
@@ -38,6 +38,6 @@ import { GridColDef } from '@material-ui/data-grid';
 | <span class="prop-name optional">type<sup><abbr title="optional">?</abbr></sup></span>              | <span class="prop-type">string</span>                                                              | <span class="prop-default">'string'<br /></span> | Type allows to merge this object with a default definition [GridColDef](/api/data-grid/grid-col-def/).          |
 | <span class="prop-name optional">valueFormatter<sup><abbr title="optional">?</abbr></sup></span>    | <span class="prop-type">(params: GridValueFormatterParams) =&gt; GridCellValue</span>              |                                                  | Function that allows to apply a formatter before rendering its value.                                           |
 | <span class="prop-name optional">valueGetter<sup><abbr title="optional">?</abbr></sup></span>       | <span class="prop-type">(params: GridValueGetterParams) =&gt; GridCellValue</span>                 |                                                  | Function that allows to get a specific data instead of field to render in the cell.                             |
-| <span class="prop-name optional">valueOptions<sup><abbr title="optional">?</abbr></sup></span>      | <span class="prop-type">(string \| { label: string; value: any })[]</span>                         |                                                  | To be used in combination with `type: 'singleSelect'`. This is an array of the possible cell values and labels. |
+| <span class="prop-name optional">valueOptions<sup><abbr title="optional">?</abbr></sup></span>      | <span class="prop-type">any[]</span>                                                               |                                                  | To be used in combination with `type: 'singleSelect'`. This is an array of the possible cell values and labels. |
```

Comment:
Why not using the same type as https://github.com/mui-org/material-ui-x/pull/2112/files#diff-011e3c058a45f4b4092aa522938afa28991cc4f218800f436d9f1b24c1c48195R77 ?

Reply:
It seems to be related to the docs generator. I'll open another PR to fix it.

Reply:
This is not a docs bug. Typing with `any` means that the object definition is useless. I updated to only allow string and numbers.